src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -41,21 +42,29 @@ def load_arguments(self, _):
         c.argument('service_runtime_subnet', options_list=['--service-runtime-subnet', '--svc-subnet'], help='The name or ID of an existing subnet in "vnet" into which to deploy the Spring Cloud service runtime. Required when deploying into a Virtual Network.', validator=validate_vnet)
         c.argument('service_runtime_network_resource_group', options_list=['--service-runtime-network-resource-group', '--svc-nrg'], help='The resource group where all network resources for Azure Spring Cloud service runtime will be created in.', validator=validate_node_resource_group)
         c.argument('app_network_resource_group', options_list=['--app-network-resource-group', '--app-nrg'], help='The resource group where all network resources for apps will be created in.', validator=validate_node_resource_group)
-
     with self.argument_context('spring-cloud update') as c:
         c.argument('sku', type=str, validator=validate_sku, help='Name of SKU, the value is "Basic" or "Standard"')
 
     for scope in ['spring-cloud create', 'spring-cloud update']:
         with self.argument_context(scope) as c:
             c.argument('app_insights_key',
-                       help="Instrumentation key of the existing Application Insights to be added for the distributed tracing",
+                       help="Instrumentation key of the existing Application Insights.",
                        validator=validate_tracing_parameters)
             c.argument('app_insights',
                        help="Name of the existing Application Insights in the same Resource Group. Or Resource ID of the existing Application Insights in a different Resource Group.",
                        validator=validate_tracing_parameters)
             c.argument('disable_distributed_tracing',
                        arg_type=get_three_state_flag(),
                        help="Disable distributed tracing, if not disabled and no existing Application Insights specified with --app-insights-key or --app-insights, will create a new Application Insights instance in the same resource group.",
+                       validator=validate_tracing_parameters,
+                       deprecate_info=c.deprecate(target='--disable_distributed_tracing', redirect='--disable_app_insights', hide=True))
+            c.argument('disable_app_insights',
+                       arg_type=get_three_state_flag(),
```

Comment:
when user specifies false, does it mean `enable'?

Reply:
yes

Reply:
`--disable-app-insights false` equals to the default behavior right? It might be better to use `action='store_true'` instead of a three state flag unless you may change the default value in the future. Three state flag is usually used for resource properties mapped to CLI options, but here it doesn't seem like the case.

Reply:
Yes, "false" is the default value. "--disable-app-insights" is to replace "--disable-distributed-tracing" so i just copy the code.  I am not sure why it use three state flag instead of “action='store_true'”